bin/upload.js

```diff
@@ -0,0 +1,29 @@
+/*
+  An S3 uploader with some useful default settings for Zooniverse static files.
+  Accepts an S3 path, relative to zooniverse-static, as its only argument.
+  Usage example: node upload.js pandora.zooniverse.org
+*/
+const Uploader = require('s3-batch-upload').default;
+
+const args = process.argv.slice(2);
+const remotePath = args[args.length - 1];
+
+async function uploadBuild() {
+  const uploader = new Uploader({
+    bucket: 'zooniverse-static',
+    localPath: './dist',
+    remotePath,
+    concurrency: '200',
+    cacheControl: {
+      '**/index.html': 'no-cache, must-revalidate',
```

Comment:
i'd prefer this was cached for a small amount of time rather than re validate at origin

we can look at adding the `s-max` header as well for shared caches (proxies / CDN), https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
I've changed it to `'max-age=60, smax-age=60'`.